### DISCUSSION TITLE
大家好，想请问下Webhook配置中变量返还的结果可以做过滤处理么？

### DISCUSSION THREAD

**Discussion by FaithAluten** [jeessy2/ddns-go#811] [Q&A] [answered]
1. `RequestBody`中`#{ipv4Result`变量会返回三个更新结果的值
2. 我只想在返回的值为`失败`的时候通知我
3. 请问有办法实现么？

- Webhook url
`https://api.telegram.org/bot{your token}/sendmessage`
- RequestBody
```
{
   "chat_id":"{notify room id}",
   "text":"域名更新结果：#{ipv4Result}"
}
```

**Comment by jeessy2** [comment, marked as answer]
没有这个功能，可以自行编译
